tests/lib/rules/no-restricted-modules.js

```diff
@@ -99,5 +101,15 @@ ruleTester.run("no-restricted-modules", rule, {
             data: { name: "foo", customMessage: "Please use 'bar' module instead." },
             type: "CallExpression"
         }]
+    }, {
+        code: "require(`fs`)",
+        options: ["fs"],
+        parserOptions: { ecmaVersion: 6 },
+        errors: [{ messageId: "defaultMessage", data: { name: "fs" }, type: "CallExpression" }]
+    }, {
+        code: "require(`foo\\\\bar`);",
```

Comment:
This is confusing looking - it requires double escaping as this is a string so actual input will be ``require(`foo\\bar`)``, but same is needed when using single/double quotes

Reply:
ops, miss-understood again ;) will change it again